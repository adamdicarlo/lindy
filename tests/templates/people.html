<h2>{{ heading }}</h2>
<ul>
  {% for person in people %}
  <li>
    {{ person.name }} ({{ person.age }}) is
    {% if person.employed %}
      working
    {% else %}
      unemployed
    {% endif %}
  </li>
  {% endfor %}
</ul>
