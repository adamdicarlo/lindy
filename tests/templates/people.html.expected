<h2>People</h2>
<ul>
  
  <li>
    Dave (25) is
    
      working
    
  </li>
  
  <li>
    John (17) is
    
      unemployed
    
  </li>
  
</ul>
